src/main/java/com/GG/T9AgeCombat/models/Unit.java

```diff
@@ -169,6 +142,16 @@ private void adjustModelCountFromPendingWounds() {
         }
     }
 
+    public List<SpecialRule> specialRuleList(List<SpecialRuleEntity> specialRuleEntities) {
+            return specialRuleEntities.stream().map(x -> SpecialRule.builder()
+                    .name(SpecialRuleEnum.valueOf(x.getName()))
+                    .limitation(LimitationEnum.valueOf(x.getLimitationValue()))
+                    .modification(ModificationEnum.valueOf(x.getModificationValue()))
+                    .timing(TimingEnum.valueOf(x.getTimingValue()))
+                    .value(x.getValue())
+                    .build()).collect(toList());
+        }
+
```

Comment:
Should this happen when we build the object so we don't have to build the list each time we request it? If not we should rename this method to `getSpecialRuleList`

Reply:
This was the attempt at trying to do the @builder override.  It didn't work since it actually shows the method as being unused.  I just forgot to delete it.  This method goes in SpecialRulesService too.

Reply:
Did you mean to leave this method in here still? I think if you were trying to override the Setter you need to follow this example https://www.baeldung.com/lombok-builder-custom-setter